2-data-model/data-modeling-in-rethinkdb.md

```diff
@@ -65,9 +65,7 @@ __Advantages of using embedded arrays:__
 
 __Disadvantages of using embedded arrays:__
 
-- Any operation on an author document requires loading all posts into
-  memory. Any update to the document requires rewriting the full array
-  to disk.
+- Any operation on a document in the `author` table requires loading all posts into memory. As well, any update to the document requires rewriting the entire document to disk. At the moment, RethinkDB does not support modifying only the single value on disk. 
```

Comment:
Regarding @chipotle's comment: We hope that we can change this at some point, but it's pretty certainly far away and might never happen. I'd remove the "At the moment". Actually I'd consider removing the last sentence (starting with "At the moment") completely. It seems redundant given the previous sentence, except that it sounds more negative.